src/Marvin/Interpolate.hs

```diff
@@ -104,10 +104,12 @@ evalExprs l = evalState (mapM stitch l) decls
     stitch :: Either a b -> S.State [c] (Either c b)
     stitch (Right str) = return $ Right str
     stitch (Left _) = do
-        (name:rest) <- get
-        put rest
-        return $ Left name
-
+        crs <- get
+        case crs of
+          [] -> error "This should never happen."
```

Comment:
In the issue #2  you said:
> I looked into it too and I think what might be cleaner here is just wrapping all user code in ()?

I'm not sure what you mean. Can you elaborate?

Reply:
I have some ideas what the problem here is. I'm suspecting that some expression like `convert - something` is being created where `- something` is the interpolated code so if it wraps with `(convert (- something))` may fix it.

I'm gonna make a few changes that either fix it or at least make it easier to debug it

Reply:
@JustusAdam oh we're talking about 2 different things here. The Fixity error isn't related to this as it turns out. I've opened a PR on the marvin repo that addresses it.